src/communication/azext_communication/generated/_help.py

```diff
@@ -0,0 +1,125 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['communication service'] = """
+    type: group
+    short-summary: communication service
+"""
+
+helps['communication service list'] = """
+    type: command
+    short-summary: "Handles requests to list all resources in a subscription."
+    examples:
+      - name: List by resource group
+        text: |-
+               az communication service list --resource-group "MyResourceGroup"
+"""
+
+helps['communication service show'] = """
+    type: command
+    short-summary: "Get the CommunicationService and its properties."
+    examples:
+      - name: Get resource
+        text: |-
+               az communication service show --name "MyCommunicationResource" --resource-group "MyResourceGroup"
+"""
+
+helps['communication service create'] = """
+    type: command
+    short-summary: "Create a new CommunicationService or update an existing CommunicationService."
+    examples:
+      - name: Create or update resource
```

Comment:
curious about why short-summary and examples description are different

Reply:
short summary refined